src/hooks/useArweave.ts

```diff
@@ -81,11 +92,49 @@ export function useArweave() {
         };
       } catch (error) {
         console.log('error', error);
-
         throw new Error(`Error fetching arweave file: ${error}`);
       }
     },
-    [arweave]
+    [arweave, onDownloadProgress]
+  );
+
+  const customGetTx = useCallback(
```

Comment:
This function contains the same code as `arweave.transactions.get(arweaveTxId)`, which is used on mainnet. Only diff is the addition of `onDownloadProgress` to the `api.get` config called on `108`.